platform/android/java/lib/src/org/godotengine/godot/GodotIO.java

```diff
@@ -55,6 +56,9 @@
 
 	MediaPlayer mediaPlayer;
 
+	WifiManager.MulticastLock multicastLock;
```

Comment:
Can you create a separate class to host the multicast logic.
For example, you could create a `NetUtils` class in the `org.godotengine.godot.utils` package. Or if you think the net related logic will grow, create a separate `net` package with a `NetManager` or `NetSocket` class to host network related code.